v2/pkg/mongo-driver/connection.go

```diff
@@ -0,0 +1,107 @@
+package mongo_driver
```

Comment:
According to idiomatic package naming we shouldn't be using underscores or mixed caps (I think the same translates to when importing packages).
https://blog.golang.org/package-names

Reply:
have changed package name to MongoDB